src/main/java/org/jabref/logic/openoffice/backend/NamedRangeReferenceMark.java

```diff
@@ -101,8 +102,8 @@ private static void createReprInDocument(XTextDocument doc,
                 : left + right;
 
         cursor.getText().insertString(cursor, bracketedContent, true);
-
-        UnoReferenceMark.create(doc, refMarkName, cursor, true /* absorb */);
+        DocumentAnnotation docAnn = new DocumentAnnotation(doc, refMarkName, cursor, true /* absorb */);
```

Comment:
For a good refactoring `true` and `false` would be replaced by an `enum` containing `ABSORB` and (probably) `KEEP`. This is out of scope of this project, just as hint.